go/integTest/glide_test_suite_test.go

```diff
@@ -96,7 +96,7 @@ func runClusterManager(suite *GlideTestSuite, args []string, ignoreExitCode bool
 			suite.T().Fatalf("Unexpected error while executing cluster_manager.py: %s", err.Error())
 		}
 
-		if exitError.Stderr != nil && len(exitError.Stderr) > 0 {
+		if len(exitError.Stderr) > 0 {
```

Comment:
Oh, so this is a new lint error fixed with the latest `staticcheck` (`S1009`)

Reply:
Yes, that's correct. The `nil` check is redundant, hence removed.